src/playground-code-editor.ts

```diff
@@ -505,21 +506,31 @@ export class PlaygroundCodeEditor extends LitElement {
       }
     });
 
-    cm.on('renderLine', (_editorInstance: Editor, line, elt) => {
-      if (!this.lineWrapping) {
-        return;
-      }
+    this._indentingLine(cm);
 
-      const basePadding = 4;
-      const charWidth = cm.defaultCharWidth();
-      const tabSize = cm.getOption('tabSize') || basePadding;
-      const off = CodeMirror.countColumn(line.text, null, tabSize) * charWidth;
+    this._codemirror = cm;
+  }
 
-      elt.style.textIndent = `-${off}px`;
-      elt.style.paddingLeft = `${basePadding + off}px`;
-    });
+  private _indentingLine(cm: CodeMirror.Editor) {
+    if (this.lineWrapping) {
+      cm.on('renderLine', this._onRenderLine);
+    } else {
+      cm.off('renderLine', this._onRenderLine);
+    }
+  }
 
-    this._codemirror = cm;
+  private _onRenderLine(
+    _editorInstance: Editor,
+    line: CodeMirror.LineHandle,
+    elt: HTMLElement
+  ) {
+    const basePadding = 4;
+    const charWidth = _editorInstance.defaultCharWidth();
+    const tabSize = _editorInstance.getOption('tabSize') || basePadding;
+    const off = CodeMirror.countColumn(line.text, null, tabSize) * charWidth;
+
+    elt.style.textIndent = `-${off}px`;
+    elt.style.paddingLeft = `${basePadding + off}px`;
   }
```

Comment:
`_` prefix on parameters usually means unused

```suggestion
  private _onRenderLine(
    editorInstance: Editor,
    line: CodeMirror.LineHandle,
    elt: HTMLElement
  ) {
    const basePadding = 4;
    const charWidth = editorInstance.defaultCharWidth();
    const tabSize = editorInstance.getOption('tabSize') || basePadding;
    const off = CodeMirror.countColumn(line.text, null, tabSize) * charWidth;

    elt.style.textIndent = `-${off}px`;
    elt.style.paddingLeft = `${basePadding + off}px`;
  }
```